code/modules/photography/photos/frame.dm

```diff
@@ -116,6 +116,24 @@
 	if(in_range(src, user))
 		framed?.show(user)
 
+/// Internal proc
+/obj/structure/sign/picture_frame/proc/try_decon(mob/living/user, obj/item/tool)
```

Comment:
```suggestion
/obj/structure/sign/picture_frame/proc/try_deconstruct(mob/living/user, obj/item/tool)
```
Try not to abbreviate too much. Prioritize clarity and don't be afraid of being a little more verbose.

Reply:
Fixed in https://github.com/tgstation/tgstation/pull/64428/commits/7a814ca8d8a41041a237da7db94c75e2cced2643